shell/server/pre-meteor.js

```diff
@@ -84,8 +84,9 @@ function wwwHandlerForGrain(grainId) {
     };
 
     if (path === "apps/index.json" ||
-        path === "apps/index-experimental.json" ||
-        path.match(/apps\/[a-z0-9]{52}[.]json/)) {
+        path.match(/apps\/[a-z0-9]{52}[.]json/) ||
+        path === "experimental/index.json" ||
+        path.match(/experimental\/[a-z0-9]{52}[.]json/)) {
```

Comment:
+1, naturally, this is sad, but if today is still "not the day" to have CORS indication in app manifest, or even as a server-wide admin setting, then OK.